packages/taquito/src/batch/rpc-batch-provider.ts

```diff
@@ -94,14 +95,16 @@ export class OperationBatch extends OperationEmitter {
 
   /**
    *
-   * @description Add a transaction operation to the batch
+   * @description Add a coantract call to the batch
```

Comment:
typo in contract

Reply:
Thanks, pushed new commit to address this